enricher/fabric8/src/main/resources/META-INF/fabric8/enricher-default

```diff
@@ -5,3 +5,4 @@ io.fabric8.maven.enricher.fabric8.IconEnricher
 io.fabric8.maven.enricher.fabric8.ExposeEnricher,1000
 io.fabric8.maven.enricher.fabric8.SpringBootHealthCheckEnricher
 io.fabric8.maven.enricher.fabric8.KarafHealthCheckEnricher
+io.fabric8.maven.enricher.fabric8.PrometheusEnricher
```

Comment:
btw, this enricher is removed by the pipeline in redhat branch so I do not thing it matters now, but should be fixed on master